tests/core/distortion/test_distortion_dexela_2923_quad.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import pytest
+
+from hexrd.core.distortion.dexela_2923_quad import Dexela_2923_quad, _find_quadrant as _find_quadrant_quad
+import hexrd.core.distortion.dexela_2923_quad as dexela_2923_quad
+
+
+@pytest.fixture(autouse=True)
+def bypass_numba_for_coverage(monkeypatch, request):
+    """When coverage is active, replace njit wrappers with their Python bodies.
+
+    This keeps the @njit decorators in the source but ensures pytest-cov
+    / Codecov see the Python lines by executing the original functions via
+    .py_func during coverage runs.
+    """
+    cov_plugin = request.config.pluginmanager.getplugin("pytest_cov")
+    cov_active = cov_plugin is not None or getattr(request.config, "cov_controller", None)
+
+    if cov_active:
+        monkeypatch.setattr(
+            dexela_2923_quad,
+            "_dexela_2923_quad_distortion",
+            dexela_2923_quad._dexela_2923_quad_distortion.py_func,
+        )
+        monkeypatch.setattr(
+            dexela_2923_quad,
+            "_dexela_2923_quad_inverse_distortion",
+            dexela_2923_quad._dexela_2923_quad_inverse_distortion.py_func
+        )
```

Comment:
Instead of having functions like this, do you think we should just use the environment variable `NUMBA_DISABLE_JIT=1` when running coverage?